docs/conf/prague/2020/welcome-wagon.rst

```diff
@@ -13,8 +13,7 @@ Write the Docs conference before.
 We are all first-timers at this year’s virtual conference. The Welcome
 Wagon is here for you, to help you make the most of it. We’d love to
 virtually meet you: feel free to reach out in Chat at the virtual
-conference if we can help
-make your time at the conference easier.
+conference if we can help make your time at the conference easier. You can also email us at {{ shortcode }}@writethedocs.org.
 
 .. contents:: All topics
```

Comment:
Feel free to ignore this, but I thought the "all topics" looked a bit weird, and maybe that heading can just go - I don't think the TOC needs an introduction.

Reply:
I agree! Actually I completely missed the "All topics" 😂 
It's gone now!